components/ResourceCreation/TestResourceCreation.tsx

```diff
@@ -115,6 +140,14 @@ function TestResourceCreation() {
     return undefined;
   };
 
+  const getConfirmationModalText = (id: string | null) => {
+    if (selectedPatient && id) {
+      const resourceIndex = currentTestCases[selectedPatient].resources.findIndex(r => r.id === id);
+      const resource = currentTestCases[selectedPatient].resources[resourceIndex];
+      return `Are you sure you want to delete ${resource.resourceType} ${getFhirResourceSummary(resource)}`;
```

Comment:
I think it would be good to include a question mark at the end of this string, but not sure if it would look weird with an ellipsis in front of it. Maybe we can play around with this and see how it would look with longer resource displays.

Reply:
![image](https://user-images.githubusercontent.com/30158156/185955056-73b32f43-68ab-4f17-93f5-8d115903d892.png)

That's a great idea! I didn't even notice that lol. So this is an example of what it looks like when the display text is really long. There are definitely some even longer ones but they are not as common. Do you think that would be okay/helpful to the user? Or is it too overwhelming?